src/common/components/header.scss

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+@import '../../common/styles/colors.scss';
+@import '../../common/styles/common.scss';
+@import '../../common/styles/fonts.scss';
+
+@mixin ellipsedText {
```

Comment:
Any chance this could be in common instead?

Reply:
As in, this looks fairly generic and I think it could go in common but I'm not sure if we have a "common" standard for this already (that is not equivalent).